README.md

```diff
@@ -11,7 +11,7 @@
 1. Check if your `Node.js` version is >= **14**.
 2. Run `npm install` to install the dependencies.
 3. replace your GEMINI api key in `.env.template` and rename the file  to `.env`
-<!-- 3. you need to either be running a local model on `http://localhost:1234` or change the openAI config in `src/utils.js` -->
+<!-- 3. you need to either be running a local model on `http://localhost:1234` or change the openAI config in `src/llm-utils.js` -->
```

Comment:
please just delete this commented line 14

Reply:
cool, done